watertap/tests/test_pure_water_pH.py

```diff
@@ -94,274 +96,298 @@
 
 __author__ = "Austin Ladshaw"
 
```

Comment:
You should probably add your name here since so much of this file has changed since I worked on it.

Also, maybe Dan's name should be here to since this file integrates EDB tests with the chemistry test.

Reply:
what do people think of doing authors like this:

    __authors__ = ["Author 1", "Author 2", etc.]
    __author__ = __authors__[0]

I feel like the single author is too restrictive, but still reasonable to expect as a first-author.

Reply:
My overengineering shoulder devil is whispering:

```py
__authors__ = {
    "Austin Ladshaw": "Concept and original implementation",
    "Ludovico Bianchi": "pytest parametrization and refactoring",
    "Dan Gunter": "EDB functionality"
}
__author__ = list(__authors__)[0]
```

, but I'm not giving in to the temptation and go with what Dan is suggesting.

Reply:
It is kind of nice to see individual contributions credited in the author list, but maybe it is overkill.

Reply:
but what about the simpler list version? I feel like we should either list the actual (multiple) authors or skip the __author__ thing altogether (borg-style).

Reply:
No, I like the `__authors__` idea, I guess more as a more human-curated `git blame` than for keeping track of individual contributions. I've added it following Dan's proposed format.